eras/byron/ledger/executable-spec/src/Byron/Spec/Ledger/STS/UTXO.hs

```diff
@@ -51,7 +51,7 @@ data UTxOEnv = UTxOEnv
   }
   deriving (Eq, Show, Generic, NoThunks)
 
-data UTxOState = UTxOState
+data ShelleyUTxOState = ShelleyUTxOState
```

Comment:
How come we are calling this `ShelleyUTxOState`  in  the byron era?

Reply:
Looks like a search and replace error. Good catch!